shared/fs/header/breadcrumb-popup.js

```diff
@@ -1,7 +1,7 @@
 // @flow
 import * as React from 'react'
 import * as Types from '../../constants/types/fs'
-import {ModalLessPopupMenu} from '../../common-adapters/popup-menu'
+import PopupMenu from '../../common-adapters/popup-menu'
```

Comment:
Looks like we are still using `PopupMenu` for now?

Reply:
Yeah until we figure out a better way to do it.

Reply:
Let's leave these classes alone for now. We do need to fix them up but let's deal with that separately